src/uct/ib/ud/accel/ud_mlx5.c

```diff
@@ -673,6 +675,19 @@ uct_ud_mlx5_ep_create(const uct_ep_params_t* params, uct_ep_h *ep_p)
     return uct_ud_mlx5_ep_t_new(params, ep_p);
 }
 
+
+static ucs_status_t
+uct_ud_mlx5_create_cq(uct_ib_iface_t *iface, uct_ib_dir_t dir,
+                      const uct_ib_iface_init_attr_t *init_attr,
+                      int preferred_cpu, size_t inl)
+{
+    uct_ud_mlx5_iface_t *mlx5_iface = ucs_derived_of(iface,
+                                                     uct_ud_mlx5_iface_t);
+
+    return uct_ib_mlx5_create_cq(iface, dir, init_attr, preferred_cpu, inl,
```

Comment:
seems like duplication of uct_rc_mlx5_create_cq

Reply:
They are different in the way of getting the enable_cqe_zipping. Anyway, this part has been reworked so I think this conversation can be closed.